src/main/kotlin/com/autonomousapps/tasks/ComputeAdviceTask.kt

```diff
@@ -310,6 +401,27 @@ internal class DependencyAdviceBuilder(
         StandardTransform(coordinates, declarations, supportedSourceSets, buildPath, isKaptApplied).reduce(usages)
       }
   }
+
+  /**
+   * Returns whether this is a KMP non-common target that can just defer to a parent common declaration in
+   * [kmpCommonDeclarations].
+   *
+   * @see GradleVariantIdentification.kmpAttribute
+   */
+  private fun Coordinates.isKmpTargetThatShouldDeferToParent(
+    targetConfiguration: String?,
+    kmpCommonDeclarations: Map<String, Set<String>>
+  ): Boolean {
+    // This only applies to module coordinates.
+    if (this !is ModuleCoordinates) return false
+    if (targetConfiguration == null) return false
+    if (isKmpTargetTarget) {
```

Comment:
Is this a typo that we're using `Target` twice in a row?

Reply:
I'm not sure what you mean?

Reply:
the name of this variable is `isKmpTargetTarget`. The word "Target" is repeated.

Reply:
![image](https://github.com/autonomousapps/dependency-analysis-gradle-plugin/assets/1361086/01bffc70-15ff-46ca-9e20-8e264d564405)

Reply:
[7a31c83](https://github.com/autonomousapps/dependency-analysis-gradle-plugin/pull/919/commits/7a31c832b6239b17feb0b3a1bee27b61835be5d5)